contracts/base/IEntityCoreFacet.sol

```diff
@@ -12,13 +13,21 @@ interface IEntityCoreFacet {
    * @param _commmissionsBP The commissions (basis points, 1 = 0.1%) to pay the broker, capital provider and Nayms
    * @param _stakeholders The three stakeholders of the policy - capital provider, insured party, broker
    */
+
```

Comment:
Pls fix the function docs here. Pls remove all the comments inside the argument list both there and everywhere else. Arguments should be documented in the function block documentation as shown in the original version of the file.